api/resources/translations/messages-ne.properties

```diff
@@ -112,9 +117,8 @@ Forms = फाराम
 From = पठाउने
 Full\ Name = पुरा नाम
 Gateway\ number = मोडेमको नम्बर
-Gateway\ number\ help = यहाँ मोडेममा राखिएको सिमको फोन नम्बर राख्नुहोला। यहि नम्बरमा रिपोर्ट तथा सन्देश पठाउनु पर्नेछ तथा यहि नम्बरबाट सन्देशहरु पठाइनेछन्।
+Gateway\ number\ help = यहाँ मोडेममा राखिएको सिमको फोन नम्बर राख्नुहोला। यहि नम्बरमा रिपोर्ट तथा सन्देश पठाउनु पर्नेछ तथा यहि नम्बरबाट सन्देशहरू पठाइनेछन्।
```

Comment:
Since other places have संदेश instead of सन्देश, shall we change this too?

Reply:
@1yuv this is the only comment I have. thanks.

Reply:
@1yuv I don't think this should be a blocker  and was only related to consistency of word. So, approving the PR.

Reply:
Hi @nsitaula , I totally missed this comment, thanks for pointing. I checked that `सन्देश` is accurate and that's in the nepali dictionary as well, so I've changed all refrences (up to 200Lines) to सन्देश.